src/components/TodoList.jsx

```diff
@@ -0,0 +1,65 @@
+import { useTodo } from "../context/TodoContext";
+import moment from "moment";
+import "./TodoList.css";
+
+export const TodoList = () => {
+  const {
+    todoList,
+    deleteTodo,
+    toggleTodo,
+    totalItems,
+    totalChecked,
+    toggleAllTodos,
+  } = useTodo();
+
+  return (
+    <div className="todo-list-container">
+      <h2>To do list:</h2>
+      {totalItems === 0 ? "🎉" :
+        <p>
+        Completed: {totalChecked} / {totalItems}
+      </p>}
+
+      <ul className="todo-ul">
+        {todoList.map((item) => {
+          return (
```

Comment:
The code for the todo list contains quite a lot of code in the end. Maybe try to break it down to even smaller component, for example for each task?